packages/amplication-server/src/models/Project.ts

```diff
@@ -0,0 +1,33 @@
+import { Workspace } from './Workspace';
+import { Resource } from './Resource';
+import { Field, ObjectType } from '@nestjs/graphql';
+
+@ObjectType({
+  isAbstract: true,
+  description: undefined
+})
+export class Project {
+  @Field(() => String, {
+    nullable: false,
+    description: undefined
+  })
+  id!: string;
+
+  @Field(() => String, {
+    nullable: false,
+    description: undefined
+  })
+  name!: string;
+
+  workspace?: Workspace;
+
+  workspaceId?: string;
+
+  @Field(() => [Resource], {
+    nullable: true,
+    description: undefined
```

Comment:
```suggestion
```

Reply:
we talked about it. All the other dto are like that. We will create an issue about it and fix it for all the entities.